pontos/nvd/cve_change_history/api.py

```diff
@@ -0,0 +1,164 @@
+# SPDX-FileCopyrightText: 2023 Greenbone AG
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from datetime import datetime, timedelta
+from types import TracebackType
+from typing import Any, AsyncIterator, Dict, Iterable, Optional, Type, Union
+
+from httpx import Timeout
+
+from pontos.errors import PontosError
+from pontos.nvd.api import (
+    DEFAULT_TIMEOUT_CONFIG,
+    NVDApi,
+    Params,
+    convert_camel_case,
+    format_date,
+)
+from pontos.nvd.models.cve_change import CVEChange, EventName
+
+__all__ = ("CVEChangeHistoryApi",)
+
+DEFAULT_NIST_NVD_CVE_HISTORY_URL = (
+    "https://services.nvd.nist.gov/rest/json/cvehistory/2.0"
+)
+
+
+class CVEChangeHistoryApi(NVDApi):
+    """
+    API for querying the NIST NVD CVE Change History information.
+
+    Should be used as an async context manager.
+
+    Example:
+        .. code-block:: python
+
+            from pontos.nvd.cve_change_history import CVEChangeHistoryApi
+
+            async with CVEChangeHistoryApi() as api:
+                async for cve_change in api.cve_changes(event_name=EventName.INITIAL_ANALYSIS):
+                    print(cve_change)
+    """
+
+    def __init__(
+        self,
+        *,
+        token: Optional[str] = None,
+        timeout: Optional[Timeout] = DEFAULT_TIMEOUT_CONFIG,
+        rate_limit: bool = True,
+    ) -> None:
+        """
+        Create a new instance of the CVE Change History API.
+
+        Args:
+            token: The API key to use. Using an API key allows to run more
+                requests at the same time.
+            timeout: Timeout settings for the HTTP requests
+            rate_limit: Set to False to ignore rate limits. The public rate
+                limit (without an API key) is 5 requests in a rolling 30 second
+                window. The rate limit with an API key is 50 requests in a
+                rolling 30 second window.
+                See https://nvd.nist.gov/developers/start-here#divRateLimits
+                Default: True.
+        """
+        super().__init__(
+            DEFAULT_NIST_NVD_CVE_HISTORY_URL,
+            token=token,
+            timeout=timeout,
+            rate_limit=rate_limit,
+        )
+
+    async def cve_changes(
```

Comment:
IMHO it's already obvious that we are querying the CVE changes from the API class name and therefore we can shorten the method name
```suggestion
    async def changes(
```

Reply:
Done in https://github.com/greenbone/pontos/pull/920/commits/1ba86c48e94cdc415e73cc3b9e8f5afcab6086cf.